tests/testthat/test-pvolfile.R

```diff
@@ -9,8 +9,14 @@ ghostfile <- paste(tempdir(), "garbage", sep = "/")
 test_that("is.pvolfile() can identify ODIM HDF5 polar volume files", {
   expect_true(is.pvolfile(pvolfile))
   expect_false(is.pvolfile(vpfile))
-  expect_warning(expect_false(is.pvolfile(textfile)))
-  expect_warning(is.pvolfile(ghostfile), "Can't find") # From get_odim_object_type
+
+  is.pvolfile(textfile) %>%
```

Comment:
If I point `textfile to a non-text file I get this, I don't think you can pipe one expect_warning() into another:
```
is.pvolfile(textfile) %>%
+   expect_warning("is not an HDF5 file") %>%
+   expect_warning("other data formats not yet recognized")
Error: `.` did not produce any warnings.
```

Reply:
Seem like you can on the latest version of `testthat` and it is also what the package maintainers now recommend in [the docs](https://testthat.r-lib.org/articles/third-edition.html)

```r
> is.pvolfile(ghostfile) %>%
+     expect_warning("Can't find") %>%
+     expect_warning("other data formats not yet recognized")
> packageVersion("testthat")
[1] ‘3.1.8’
```

Reply:
I think in the latest versions you can, but the function def still has bugs.  For example, these work:

```r
> packageVersion("testthat")
[1] ‘3.1.8’
> is.pvolfile(textfile) %>% expect_warning("is not an HDF5 file") %>% expect_warning('other data formats')
> is.pvolfile(ghostfile) %>%
+     expect_warning("Can't find") %>%
+     expect_warning("other data formats not yet recognized")
```